wordpress-mobile/WordPress-iOS
# Use new colors in Login/Signup screens

Refs #11683

This fixes colors in the signup/login screen to use the new palette.

<img width="458" alt="image" src="https://user-images.githubusercontent.com/517257/61333971-e1d9af80-a7e5-11e9-898a-61d3157a620c.png">


Separate PR for the authenticator pod: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/107

To test:
- log out
- go through the login process
- log out again
- signup to a new account
- ensure all the old nasty colors are gone

Update release notes:
